pkg/console/operatorclient/operatorclient.go

```diff
@@ -0,0 +1,61 @@
+package operatorclient
+
+import (
+	"k8s.io/client-go/tools/cache"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
+	operatorv1informers "github.com/openshift/client-go/operator/informers/externalversions"
+	"github.com/openshift/console-operator/pkg/api"
+)
+
+type OperatorClient struct {
+	Informers operatorv1informers.SharedInformerFactory
+	Client    operatorv1client.ConsolesGetter
+}
+
+func (c *OperatorClient) Informer() cache.SharedIndexInformer {
+	return c.Informers.Operator().V1().Consoles().Informer()
+}
+
+func (c *OperatorClient) GetOperatorState() (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
```

Comment:
If they're not being used, why did we copy them in?

Reply:
We copied the file & tweaked it like others:
https://github.com/search?q=org%3Aopenshift+OperatorClient&type=Code
There are about 73 other versions of this.

This is an interface impl, we need these `funcs` whenever we pass the operator client around.

Reply:
The 4th arg to `clusterOperatorStatus := status.NewClusterOperatorStatusController( )` line `125` of `starter.go` requires [this interface](https://github.com/openshift/library-go/blob/master/pkg/operator/v1helpers/interfaces.go#L8):

```golang
// https://github.com/openshift/library-go/blob/master/pkg/operator/v1helpers/interfaces.go#L8
type OperatorClient interface {
	Informer() cache.SharedIndexInformer
	GetOperatorState() (spec *operatorv1.OperatorSpec, status *operatorv1.OperatorStatus, resourceVersion string, err error)
	UpdateOperatorSpec(string, *operatorv1.OperatorSpec) (spec *operatorv1.OperatorSpec, resourceVersion string, err error)
	UpdateOperatorStatus(string, *operatorv1.OperatorStatus) (status *operatorv1.OperatorStatus, err error)
}
```

Reply:
Got it, thanks. Just making sure this wasn't dead code.